compiler/rustc_metadata/src/errors.rs

```diff
@@ -24,6 +24,14 @@ pub struct LibRequired<'a> {
     pub kind: &'a str,
 }
 
+#[derive(Diagnostic)]
+#[diag(metadata_rustc_lib_required)]
+#[help]
```

Comment:
@davidtwco why is this `#[help]` attribute required? Can we remove that requirement somehow? We already have `.help` in the .ftl file, I wouldn't expect both to be necessary.